code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+	icon_state = "mauler_mag-1"
+	base_icon_state = "mauler_mag"
+	ammo_type = /obj/item/ammo_casing/c9mm
+	caliber = "9mm"
+	max_ammo = 12
+
+/obj/item/ammo_box/magazine/m9mm_mauler/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state]-[!!ammo_count()]"
+
+/obj/item/gun/ballistic/automatic/pistol/spitter
+	name = "\improper Spitter"
+	desc = "An Old Frontiersmen machine pistol. While its origin is controversial, and it is primarily associated with criminals, it is widely cloned and used in the frontier. Chambered in 9mm."
+	icon = 'icons/obj/guns/manufacturer/frontier_import/48x32.dmi'
+	lefthand_file = 'icons/obj/guns/manufacturer/frontier_import/lefthand.dmi'
+	righthand_file = 'icons/obj/guns/manufacturer/frontier_import/righthand.dmi'
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "spitter"
+	item_state = "spitter"
+	mag_type = /obj/item/ammo_box/magazine/spitter_9mm
+	bolt_type = BOLT_TYPE_OPEN
+	weapon_weight = WEAPON_LIGHT
+	show_magazine_on_sprite = TRUE
+	manufacturer = MANUFACTURER_IMPORT
+
+	spread = 20
+	spread_unwielded = 35
+	dual_wield_spread = 35
+	wield_slowdown = 0.25
+	wield_delay = 0.2 SECONDS
+	fire_delay = 0.09 SECONDS
+
+	fire_sound = 'sound/weapons/gun/smg/spitter.ogg'
+	rack_sound = 'sound/weapons/gun/smg/spitter_cocked.ogg'
+	rack_sound_vary = FALSE
+
+	load_sound_vary = FALSE
+	eject_sound_vary = FALSE
+	load_sound = 'sound/weapons/gun/smg/spitter_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/smg/spitter_reload.ogg'
+	eject_sound = 'sound/weapons/gun/smg/spitter_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/smg/spitter_unload.ogg'
+
+	valid_attachments = list(
+		/obj/item/attachment/silencer,
+		/obj/item/attachment/foldable_stock/spitter
+	)
+
+	slot_available = list(
+		ATTACHMENT_SLOT_MUZZLE = 1,
+		ATTACHMENT_SLOT_STOCK = 1
+	)
+	slot_offsets = list(
+		ATTACHMENT_SLOT_MUZZLE = list(
+			"x" = 32,
+			"y" = 23,
+		),
+		ATTACHMENT_SLOT_STOCK = list(
+			"x" = -5,
+			"y" = 18,
+		)
+	)
+
+	default_attachments = list(/obj/item/attachment/foldable_stock/spitter)
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+
+
+/obj/item/ammo_box/magazine/spitter_9mm
+	name = "spitter pistol magazine (9mm)"
+	desc = "A thin, 30-round magazine for the spitter machine pistol. These rounds do okay damage, but struggle against armor."
+	icon_state = "spitter_mag-1"
+	base_icon_state = "spitter_mag"
+	ammo_type = /obj/item/ammo_casing/c9mm
+	caliber = "9mm"
+	max_ammo = 30
+
+/obj/item/ammo_box/magazine/spitter_9mm/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state]-[!!ammo_count()]"
+
+
+/obj/item/gun/ballistic/automatic/smg/pounder
+	name = "Pounder"
+	desc = "A strange frontiersmen weapon. With a high ammo count and a low caliber, this gun makes up for its lack of power with it's extremely high rate of fire, hence the name. Chambered in .22 LR."
```

Comment:
"An unusual submachine gun of Frontiersman make. A miniscule cartridge lacking both stopping power and armor penetration is compensated for with best-in-class ammunition capacity and cycle rate. Chambered in .22 LR."